app/lib/brexit_checker/results/result_presenter.rb

```diff
@@ -0,0 +1,39 @@
+class BrexitChecker::Results::ResultPresenter
+  attr_reader :criteria_keys
+
+  def initialize(criteria_keys)
+    @criteria_keys = criteria_keys
+  end
+
+  def criteria
+    BrexitChecker::Criterion.load_by(criteria_keys)
+  end
+
+  def actions
+    filtered = BrexitChecker::Action.load_all.select do |a|
+      a.show?(criteria_keys)
+    end
+    sorted_actions(filtered)
+  end
+
+  def sorted_actions(actions)
```

Comment:
can this be private?

Reply:
I made a couple more methods private as well. I've added as a fresh commit. If it looks ok to you I will fixup the commit.